omnibus/config/software/postgresql13.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "postgresql13"
+
+default_version "13.4"
```

Comment:
We should target 13.5 now: https://www.postgresql.org/docs/13/release-13-5.html

Reply:
@tas50 we initially had the upgrade tested with 13.3. We had already tested it for 13.4 again. So to change this again we need to perform the testing again. Can we have it released with 13.4 now and any next release can have the latest minor version upgraded which will not be a challenge but every time testing the data migration between 2 major version of PG takes considerable amount of time.

Reply:
Long term we're going to need to figure out how to automate that testing similar to what is done with Infra Server. New PostgreSQL releases come out probably every 6 weeks and those almost always include CVE fixes. We need to be able to quickly pull those into the client and release them within a few days. 13.5 includes 2 CVE fixes so while we can ship with 13.4, we will want to followup shortly afterwards with the 13.5 upgrade.

Reply:
Totally makes sense. The minor versions can actually be auto updated once we are done with this major version upgrade. This major version upgrade needs a specific software definition of PG in place. But once the major version upgrade is done we should be good with just specifying PG version 13 everywhere irrespective of the minor version currently in circulation because that doesn't need to run the pg_upgrade command which belongs to a very specific version of PG. Hope this makes sense. We will definitely make the process smoother @tas50 

@saghoshprogress pls note this. we need to include this in our backlog to handle PG major version only in the upcoming release of supermarket after this major version upgrade.